WolvenKit.Common/Services/HashService.cs

```diff
@@ -206,6 +206,10 @@ private void ReadTweakNames(Stream memoryStream)
             TweakDBIDPool.SetNative(dict);
         }
 
+        public virtual bool AddResourcePath(string resourcePath) => false;
+
+        public virtual bool AddTweakName(string tweakName) => false;
```

Comment:
Why adding that here?
Its currently not supported to add custom ResourcePath/TweakNames in anything except Wkit GUI.

Reply:
If it's not in there, I can't have the DocumentTools under Tools, where they have to be to find other dependencies... it's turtles all the way down, I'm afraid

Reply:
Oh, didn't saw that you placed `DocumentTools` in `ModKit`.
Still not convinced of the usefullness of it there.
Looks like an class which is meant to be used as part of GUI (Project context) in a namespace which shouldn't really know the GUI / has projects.

Reply:
moved it out :)